src/content/pages/index.md

```diff
@@ -2,32 +2,148 @@
 template: home-page
 path: /
 hero:
-  title: Threshold powers user sovereignty on the public blockchain.
-  body: Access cryptographic tools that ensure full control over your digital assets.
+  title: Full control over your digital assets
+  body: Use threshold cryptographic tools that power user sovereignty on the
+    blockchain.
   ctaButtons:
-    - label: Start Staking
+    - label: Launch Dapp
       url: https://dashboard.threshold.network
-    - label: Read the Blog
-      url: https://blog.threshold.network/
-summary:
-  title: Why Threshold Network?
-  body: Threshold Network is the first ever on-chain merge between two existing
-    networks. <a href="https://blog.threshold.network/" target="_blank"
-    rel="noopener noreferrer" >Learn more</a>
-  cards:
+    - label: Join Discord
+      url: https://discord.gg/VafbhzJ3vW
+stakerRole:
+  title: Threshold Staking
+  description:
+    Stakers escrow T tokens to run a node on the Threshold Network and
+    earn rewards.
+  image: /images/staking-threshold.png
+  buttons:
+    - label: Start Staking
+      url: https://dashboard.threshold.network/overview/network
+      variant: EXTERNAL_SOLID
+    - variant: INTERNAL_OUTLINE
+      label: Learn More
+      url: /earn/staker
+  rowReverse: false
+  bgColor: "#858585"
+lpRole:
+  title: Provide Liquidity
+  description: Liquidity providers earn yields by depositing assets into liquidity pools.
+  image: /images/liquidity.png
+  buttons:
+    - label: Learn More
+      url: /earn/liquidity-provider
+      variant: INTERNAL_SOLID
+  rowReverse: true
+  bgColor: "#858585"
+btcRole:
+  title: Bridge BTC to Ethereum
+  description: tBTC is Threshold’s decentralized bridge to bring BTC to the
+    Ethereum network.
+  image: /images/tbtc.png
+  buttons:
+    - label: Mint tBTC
+      url: NEED_URL
+      variant: EXTERNAL_SOLID
+    - label: Learn More
+      url: /earn/btc
+      variant: INTERNAL_OUTLINE
+  rowReverse: false
+  bgColor: "#333333"
+tokenHolderRole:
+  title: Token Holder DAO
+  description: Make the most of your T tokens on the Threshold Network by
+    participating in DAO governance.
+  image: /images/token-holder.png
+  buttons:
+    - label: Governance Forum
+      url: NEED_URL
+      variant: EXTERNAL_SOLID
+    - label: Learn More
+      url: /earn/token-holder
+      variant: INTERNAL_OUTLINE
+  rowReverse: true
+  bgColor: "#3c3c3c"
+migrationInfo:
+  title: Do you own KEEP or NU?
+  description: Keep and NuCypher merged to form the Threshold Network. Migrate
+    your tokens to T!
+  image: /images/legacy-tokens.png
+  buttons:
+    - label: Migrate tokens
+      url: https://dashboard.threshold.network/
+      variant: EXTERNAL_OUTLINE
+  bgColor: "#141414"
+  rowReverse: false
+harnessThePower:
```

Comment:
In this section, we don't have bg color picker. Wonder if we really need bg color picker in cms. We should ask our designers.

Reply:
I removed it. They never asked for it, just figured it would be helpful